apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/service/NamespaceServiceIntegrationTest.java

```diff
@@ -62,6 +71,11 @@ public class NamespaceServiceIntegrationTest extends AbstractIntegrationTest {
   private ReleaseHistoryService releaseHistoryService;
   @Autowired
   private InstanceConfigRepository instanceConfigRepository;
+  @Autowired
+  private NamespaceRepository namespaceRepository;
+
+  @Mock
+  private BizConfig bizConfig;
```

Comment:
Can we use `MockBean` here instead of setting the field using ReflectionTestUtils?